foundry.toml

```diff
@@ -1,5 +1,5 @@
 [profile.default]
-solc = "0.8.18"
+solc = "0.8.20"
```

Comment:
is slither going to complain about this new version?

Reply:
oh, you're right, i forgot Slither would complain, I'm reverting back to `0.8.18`